hardware_accelerators/gaudi-ai-accelerator.adoc

```diff
@@ -0,0 +1,22 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="gaudi-ai-accelerator"]
+= Intel Gaudi AI accelerators
+include::_attributes/common-attributes.adoc[]
+:context: gaudi-ai-accelerator
+
+toc::[]
+
+:FeatureName: Intel Gaudi AI accelerators
+
+You can use Intel Gaudi AI accelerators for your {product-title} generative AI and machine learning (AI/ML) applications. Intel Gaudi AI accelerators offer a cost-efficient, flexible, and scalable solution for optimized deep learning workloads.
+
+Red{nbsp}Hat supports Intel Gaudi 2 and Intel Gaudi 3 devices. Intel Gaudi 3 devices provide significant improvements in training speed and energy efficiency.
+
+include::modules/gaudi-ai-accelerator-prerequisites.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* link:https://docs.habana.ai/en/latest/Installation_Guide/Additional_Installation/OpenShift_Installation/index.html[OpenShift Installation].
+
+* link:https://docs.redhat.com/en/documentation/red_hat_openshift_ai_self-managed/2.20/html/working_with_accelerators/intel-gaudi-ai-accelerator-integration_accelerators#intel-gaudi-ai-accelerator-integration_accelerators[Intel Gaudi AI Accelerator integration].
```

Comment:
Somehow missed this earlier, but I would remove the periods from these links, and also show that the first link leads to external docs:
```suggestion
* link:https://docs.habana.ai/en/latest/Installation_Guide/Additional_Installation/OpenShift_Installation/index.html[OpenShift Installation] (Intel Gaudi documentation)

* link:https://docs.redhat.com/en/documentation/red_hat_openshift_ai_self-managed/2.20/html/working_with_accelerators/intel-gaudi-ai-accelerator-integration_accelerators#intel-gaudi-ai-accelerator-integration_accelerators[Intel Gaudi AI Accelerator integration]
```